crates/transaction-workloads-lib/src/args.rs

```diff
@@ -351,6 +353,10 @@ impl TransactionTypeArg {
             },
             TransactionTypeArg::DeserializeU256 => call_custom_module(EntryPoints::DeserializeU256),
             TransactionTypeArg::SimpleScript => call_custom_module(EntryPoints::SimpleScript),
+            TransactionTypeArg::PermissionedTransfer => {
+                call_custom_module(EntryPoints::APTPermissionedTransfer)
```

Comment:
nit: would be nice to have consistency `PermissionedTransfer` -> `EntryPoints::PermissionedTransfer` or rename `PermissionedTransfer` to `AptPermissionedTransfer`